packages/worker/src/workflow/workflow-worker-thread.ts

```diff
@@ -23,6 +23,26 @@ function ok(requestId: BigInt): WorkerThreadResponse {
 let workflowCreator: VMWorkflowCreator | undefined;
 const workflowByRunId = new Map<string, VMWorkflow>();
 
+// Best effort to catch unhandled rejections from workflow code.
+// We crash the thread if we cannot find the culprit.
+process.on('unhandledRejection', (err, promise) => {
```

Comment:
To confirm, this is run from the outside not the inside, right? Is there any way to catch an unhandled rejection _inside_ the VM or add such a handler to just the VM context only?